src/components/AssetSearch/AssetList.tsx

```diff
@@ -20,6 +25,8 @@ type ItemData<T> = {
 }
 
 export const AssetList = ({ assets, handleClick }: AssetListProps) => {
+  const rowData = useSelector(selectPortfolioAccountRows)
```

Comment:
If you name this one `portfolioAcountRows` (plural) as per the selector name suggests, you won't have issues to name the current item `portfoioAccountRow` (singular) in here: 
`rowData.find(d => d.assetId === asset.assetId)?.fiatAmount`